spring-pulsar-dependencies/build.gradle

```diff
@@ -21,10 +21,10 @@ ext {
 	protobufJavaVersion = '3.21.5'
 	pulsarTestcontainersVersion = '1.17.3'
 	pulsarVersion = '2.10.1'
-	reactorVersion = '2020.0.17'
-	springBootVersion = '3.0.0-SNAPSHOT'
+	reactorVersion = '2022.0.0-M4'
```

Comment:
We were really off wrt to the Reactor BOM version (2020?) but all good now. SB and SF both use reactor `3.5.0-M4 `and that is contained in `2022-0.0-M4`.